### DISCUSSION TITLE
LXQt appearance: Window buttons/border

### DISCUSSION THREAD

**Discussion by HighTechYong** [lxqt/lxqt#2515] [Q&A] [answered]
Hello, dear LXQt community! I just wanna know how to change my windows border buttons, i mean (close button, min/max window) and window border itself. I use Debian and my version of LXQt is 1.2.0. My WM is XFWM4. 
1. Can I do this with qt5 kvantum? If so, should i use qss? 
2. Can I do this without qt5 kvantum and use only fusion and qss?

**Comment by stefonarch** [comment, marked as answer]
Window buttons and -themes are part of the WM, you need tweaking/changing xfwm4 themes.
